src/frontend/screens/Settings/Experiments/P2pUpgrade.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { StyleSheet, Text, View } from "react-native";
+import { NavigationStackScreenComponent } from "react-navigation-stack";
+import SettingsContext from "../../../context/SettingsContext";
+import { LIGHT_GREY } from "../../../lib/styles";
+import MaterialIcon from "react-native-vector-icons/MaterialIcons";
+import HeaderTitle from "../../../sharedComponents/HeaderTitle";
```

Comment:
nit: fix sorting of imports here so that npm modules are first, followed by relative imports